app/src/main/java/co/techmagic/hr/presentation/ui/manager/quotes/QuotesManagerImpl.kt

```diff
@@ -0,0 +1,42 @@
+package co.techmagic.hr.presentation.ui.manager.quotes
+
+import android.content.Context
+import co.techmagic.hr.R
+import java.util.*
+
+
+class QuotesManagerImpl(val context: Context) : QuotesManager {
```

Comment:
`Impl` does not really mean anything. I suggest `AndroidResQuotesManager` which makes it clear that it is android  resources

Reply:
Done